packages/settings-library/tests/test_utils_cli.py

```diff
@@ -98,35 +119,41 @@ def test_compose_commands(cli: typer.Typer, cli_runner: CliRunner):
     result = cli_runner.invoke(cli, ["settings", "--help"], catch_exceptions=False)
     print(result.stdout)
 
-    assert "--compact" in result.stdout
+    assert ("--compact" in result.stdout) or (
+        "\x1b[1;36m-\x1b[0m\x1b[1;36m-compact" in result.stdout
+    )
```

Comment:
@pcrespov I am trying to find a reason why this is happening. Locally I don't get formatted text, while in the CI the test is formatted.

This started happening after installing `rich` which is required by `rich.print`. Any hints on what one could do to remove the formatting of rich in this settings? I have looked at the options for Typer() and CliRunner() but nothing I have tried helped.
As you can see in this module. Unloading rich before importing it does not help as well. I'm running out of ideas.